generate_refresh_token.py

```
@@ -0,0 +1,60 @@
+import os
+import json
+from google.auth.transport.requests import Request
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.credentials import Credentials
```

Ruff (0.8.2)

`google.auth.credentials.Credentials` imported but unused

Remove unused import: `google.auth.credentials.Credentials`

(F401)